### PR TITLE
Add excluding of nonValid users to group script

### PR DESCRIPTION
**Pull request by stavamichal**
- add req attribute A_M_core_status (needed for checking statuses)
 - add req attribute A_F_def_excludeNonValidUsersFromUnixGroups
   (attribute with boolean type value, if eq to "true", then use only
   valid members)
 - add logic to generating script "group" for excluding nonValid users if needed

### CODE REVIEW

File: perun-services/gen/group [hunk 1]
```diff
@@ -25,24 +25,31 @@ my $DIRECTORY = perunServicesInit::getDirectory;
...
 our $STRUC_USERS;        *STRUC_USERS = \1;
 our $STRUC_SYSTEM;       *STRUC_SYSTEM = \2;
 our $groupStruc = {};     #$groupStruc->{$groupName}->{$STRUC_GID} = $gid;         # gid of group $groupname
-													#$groupStruc->{$groupName}->{$STRUC_USERS}->{$login} = 1;  #this mean user exists in group
-													#$groupStruc->{$groupName}->{$STRUC_SYSTEM} = 1;         # group is system group
+                          #$groupStruc->{$groupName}->{$STRUC_USERS}->{$login} = 1;  #this mean user exists in group
+                          #$groupStruc->{$groupName}->{$STRUC_SYSTEM} = 1;         # group is system group
+
+my %facilityAttributes = attributesToHash $data->getAttributes;
+our $excludeNonValidUsers = $facilityAttributes{$A_FACILITY_EXCLUDE_NON_VALID_USERS};
```
Comment (licehammer) [inline perun-services/gen/group]: s/our/my/

**Comment by stavamichal** [comment]
Fixed.

File: perun-services/gen/group [hunk 2]
```diff
@@ -154,6 +160,13 @@ sub processMembersData {
 	my ($membersElement, $groupName) = @_;
 	for my $memberData ($membersElement->getChildElements) {
 		my %memberAttributes = attributesToHash $memberData->getAttributes;
+
```
Comment (licehammer) [inline perun-services/gen/group]: Trailing whitespace. Please fix it.

**Comment by zlamalp** [comment]
Changes required for deployment are already on devel, production and all virtual appliances. Can we merge it ?

**Comment by licehammer** [comment]
Go ahead.

**Pull request merged**